mappings/net/minecraft/data/client/model/TexturedModel.mapping

```diff
@@ -0,0 +1,52 @@
+CLASS net/minecraft/class_4946 net/minecraft/data/client/model/TexturedModel
+	FIELD field_23036 CUBE_ALL Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23037 CUBE_MIRRORED_ALL Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23038 CUBE_COLUMN Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23039 CUBE_COLUMN_HORIZONTAL Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23040 CUBE_BOTTOM_TOP Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23041 CUBE_TOP Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23042 ORIENTABLE Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23043 ORIENTABLE_WITH_BOTTOM Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23044 CARPET Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23045 TEMPLATE_GLAZED_TERRACOTTA Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23046 CORAL_FAN Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23047 PARTICLE Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23048 TEMPLATE_ANVIL Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23049 LEAVES Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23050 TEMPLATE_LANTERN Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23051 TEMPLATE_HANGING_LANTERN Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23052 TEMPLATE_TORCH Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23053 TEMPLATE_TORCH_WALL Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23054 TEMPLATE_SEAGRASS Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23055 END_FOR_TOP_CUBE_COLUMN Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23056 END_FOR_TOP_CUBE_COLUMN_HORIZONTAL Lnet/minecraft/class_4946$class_4947;
+	FIELD field_23057 WALL_CUBE_BUTTOM_TOP Lnet/minecraft/class_4946$class_4947;
```

Comment:
Are these taken from IDs? If not, they should be in the right order (`TEMPLATE_TORCH` -> `TORCH_TEMPLATE`, for example).

Reply:
![image](https://user-images.githubusercontent.com/7806504/75952573-b985ea80-5e74-11ea-9133-b64deeb5bae8.png)
I am trying to fit with those inferred names.